### ISSUE TITLE
Email Access Permissions

### ISSUE THREAD

**Issue by igaralf** [pawp81/AdminSubmissionsAPI#3]
Hello
In order fo the latest script to work, which email related permissions must the Graph app have?

**Comment by pawp81** [comment]
Good catch. I didn't go live with the new scripts to submit email messages yet :-)
As described in the Readme.md for email submission:

- Read user mail (Mail.Read)
- Read user and shared mail (Mail.Read.Shared) (optional - needed if authenticated user should be able to submit emails not only for his/her mailbox but also from shared mailbox she/he has access to). Script cannot run as a daemon. It requires authentication of the user performing the submission.

https://github.com/pawp81/AdminSubmissionsAPI/blob/master/EmailSubmission.md

Let me know if it helps?
